src/edu/cmu/cs/sasylf/util/ErrorReport.java

```diff
@@ -20,6 +20,8 @@ public String formatMessage() {
 		else msg = getSpan().getLocation().toString() + ": ";

```

Comment:
Have you checked that it makes sense to mark errors instead of warnings?  If so, this seems like a good change.  But I don't want to see "error: Error" ...